R/transformCounts.R

```diff
@@ -131,20 +170,21 @@ setGeneric("transformAssay", signature = c("x"),
 #' @export
 setMethod("transformAssay", signature = c(x = "SummarizedExperiment"),
     function(x,
-             assay.type = "counts", assay_name = NULL,
-             method = c("alr", "chi.square", "clr", "frequency", "hellinger",
-                        "log", "log10", "log2", "max", "normalize", "pa",
-                        "range", "rank", "rclr", "relabundance", "rrank",
-                        "standardize", "total", "z"),
-             MARGIN = "samples",
-             name = method,
-             pseudocount = FALSE,
-             ...){
+        assay.type = "counts", assay_name = NULL,
+        method = c("alr", "chi.square", "clr", "css", "css.fast", "frequency", 
```

Comment:
"css.fast" should probably be "css_fast". For instance, we have in alpha diversity indices "gini_dominance"

Reply:
ACK.